src/main/java/com/google/sps/utils/StringConstants.java

```diff
@@ -31,6 +33,10 @@
   public static final String SUMMARY_SCORE = "summaryScore";
   public static final String TIMESTAMP_PROPERTY = "timestamp";
   public static final String VALUE = "value";
+  public static final String CHAT_PAGE_NAMESPACE = "templates.chat.chatPage";
+  public static final String KEYS_TXT_FILE = "keys.txt";
+  public static final String CHAT_ERROR_NAMESPACE = "templates.chat.error";
+  public static final String REFERER_HEADER = "Referer";
```

Comment:
look at Dylan's file for ordering, he put all the template names at the top and keys and referer should be in alphabetical order in the chat constants section